hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -218,17 +218,6 @@
       This volume choosing policy randomly chooses two volumes with remaining space and then picks the one with lower utilization.
     </description>
   </property>
-  <property>
```

Comment:
Please keep this in ozone-default.xml.  Also we should add the other property here too. I prefer a centralized place ozone-default.xml over looking around the code base to find the property, which is not very friendly for normal Ozone users.

Reply:
Thanks @ChenSammi for the review.

We should not duplicate property between `ozone-default.xml` and annotated config objects.  While looking at `ozone-default.xml` is convenient, searching (not looking) for property in code is not much more difficult (find in file vs. find in project).  Both kinds of properties are listed in `/config` and `/conf` endpoints.  (I've just found that descriptions are missing, also for both kind of properties.  Filed HDDS-12656.)

Config objects have several benefits, which were described in the design doc accepted several years ago (HDDS-1466).  We also try to further improve them (see HDDS-12424 as the most recent example).